src/App.jsx

```diff
@@ -1,8 +1,41 @@
-import { useState } from 'react'
 import './App.css'
+import { useState, useEffect } from 'react'
+import { Route, Routes } from 'react-router-dom'
+import Dashboard from './pages/Dashboard'
+import PersonProfile from './pages/PersonProfile'
 
 export default function App() {
   const [hiredPeople, setHiredPeople] = useState([])
+  const [people, setPeople] = useState([])
+
+  const addToHired = (data) =>
+  {
+    // Sees if the person is already in list (has been edited)
+    let found = false
+    for (let i = 0; i < hiredPeople.length; i++)
+    {
+      if (hiredPeople[i] === data.person)
+      {
+        console.log("NOT NEW PERSON!")
+        found = true
```

Comment:
probably best to break the loop after finding the person you are looking for, to prevent unnecessary iterations